doc/v2/howto/cmd_parameter/index_en.rst

```diff
@@ -2,10 +2,25 @@
 
 Set Command-line Parameters
 ===========================
+The implementation of the deep learning algorithm has a variety of characteristics，such as running environment, running stage, structure of the model and the traning strategy. PaddlePaddle supports the user to set various command-line parameters flexibly, which help to achieve control of the model training or prediction process.
+
+In this part，we take several actual scenes as an example, and the use of some command-line parameters is displayed:
+
+..  toctree::
+  :maxdepth: 1
+
+  use_case_cn.md
+
+Then, we summarize and classify the use of all command-line parameters:
+
+..  toctree::
+  :maxdepth: 1
+
+  arguments_cn.md
+
+Finally, the detailed descriptions are given, and we try to explain the propeties and significance of these command-line parameters in detail:
 
 ..  toctree::
   :maxdepth: 1
 
-  use_case_en.md
-  arguments_en.md
-  detail_introduction_en.md
+  detail_introduction_cn.md
```

Comment:
This should be detail_introduction_en.md

Reply:
Hi @abhinavarora, would you please tell me more detail about 'it would be great if you could update the URLs also.' I am not clear about the URLs.

Reply:
Hi @xiangjinxin1019 Please ignore that comment. I was referring to names like `use_case_en.md`